source/atlas-data-lake-testing/tests/crud/listDatabases.yml

```diff
@@ -0,0 +1,13 @@
+tests:
```

Comment:
Note: this file can omit `database_name` and `collection_name`, since it won't be used for anything.